meinberlin/assets/scss/_variables.scss

```diff
@@ -11,10 +11,17 @@ $body-bg: #fff;
 $bg-secondary: #f6f6f6;
 $text-color: #222;
 $text-color-inverted: $body-bg;
+$text-color-introvert: lighten($text-color, 15%);
+$text-color-extrovert: darken($text-color, 10%);
```

Comment:
I would prefer if we could do without these for now. We can add more colors once the designs are ready.